src/experiments.jl

```diff
@@ -7,7 +7,8 @@ Creates an MLFlow experiment.
 - `mlf`: [`MLFlow`](@ref) configuration.
 - `name`: experiment name. If not specified, MLFlow sets it.
 - `artifact_location`: directory where artifacts of this experiment will be stored. If not specified, MLFlow uses its default configuration.
-- `tags`: a Dictionary of key-values which tag the experiment.
+- `tags`: a Vector of Dictionaries which tag the experiment.
```

Comment:
@pebeto Is this really the way this works?

Reply:
Hi @ablaom. Yes, `tags` must be a Vector of Dictionaries with key and value, as expected in `mlflow` (i.e. https://mlflow.org/docs/latest/rest-api.html?highlight=tags#experimenttag).

Reply:
Also is good to note that I'm working on the next big change for this library (https://github.com/JuliaAI/MLFlowClient.jl/commit/9fe5e9ae65ae9760e013d80c3c696038ef9a495d), where I'm pairing everything with the last `mlflow` specification (including types).

Reply:
This just seems weird. If I want to assign the value "bar" to a tag called "foo", and a value "gala" to a tag called "missy", I would have expected to pass `Dict("foo" => "bar", "missy" => "gala")`. But my reading here is that I should pass `[Dict("key" => "foo", "value" => "bar"), Dict("key" => "missy", "value" => "gala")]` instead?

Reply:
I also found it counterintuitive, which is why I opened the pull request.

Reply:
Right. Then I'll leave to @pebeto to finalise the review. Perhaps we can make this more intuitive at the next breaking release.